server/chat-plugins/tcgtabletop.ts

```diff
@@ -38,39 +39,46 @@ async function getFandom(site, pathName, search) {
 	return json;
 }
 
-async function searchFandom(site, query) {
+async function searchFandom(site: string, query: string) {
 	const result = await getFandom(site, SEARCH_PATH, {query, limit: 1});
 	if (!Array.isArray(result.items) || !result.items.length) throw new Error(`Malformed data`);
 	if (!result.items[0] || typeof result.items[0] !== 'object') throw new Error(`Malformed data`);
 	return result.items[0];
 }
 
-async function getCardDetails(site, id) {
-	const result = await getFandom(site, DETAILS_PATH, {ids: id, abstract: 0, width: 80, height: 115});
+async function getCardDetails(site: string, id: string) {
+	const specifications = {
+		ids: id,
+		abstract: 0,
+		width: 80,
+		height: 115,
+	};
+
+	const result = await getFandom(site, DETAILS_PATH, specifications);
 	if (typeof result.items !== 'object' || !result.items[id] || typeof result.items[id] !== 'object') {
 		throw new Error(`Malformed data`);
 	}
 	return result.items[id];
 }
 
-exports.commands = {
+export const commands: ChatCommands = {
 	ygo: 'yugioh',
 	yugioh(target, room, user) {
 		if (!this.canBroadcast()) return;
 		if (room.roomid !== 'tcgtabletop') return this.errorReply("This command can only be used in the TCG & Tabletop room.");
-		let subdomain = 'yugioh';
-		let query = target.trim();
+		const subdomain = 'yugioh';
+		const query = target.trim();
 		if (!query) return this.parse('/help yugioh');
 
-		return searchFandom(subdomain, query).then(data => {
+		return searchFandom(subdomain, query).then((data: { url: any, title: any, id: any}) => {
```

Comment:
I'm surprised this isn't a style error, but there should be no space in front of `url`.

Also, these can all be `unknown`.

```suggestion
		return searchFandom(subdomain, query).then((data: {url: unknown, title: unknown, id: unknown}) => {
```